### PR TITLE
[81X] Primary Vertex Validation: enable choice of y-axis range from all-in-one tool

### PR DESCRIPTION
**Pull request by mmusich**
This PR adds few plotting options parameters in the all-in-one tool, to allow explicit choice of y-axis limits in the Primary Vertex Validation.
It should help in the determination of the IOVs for the 2016 EOY legacy alignment.

### CODE REVIEW

**Comment by mmusich** [comment]
please test

File: Alignment/OfflineValidation/macros/FitPVResiduals.C [hunk 1]
```diff
@@ -43,13 +43,148 @@
...
+	     _w_dxyPhiMax(120.),    _w_dzPhiMax(180.),      
+	     _w_dxyEtaMax(120.),    _w_dzEtaMax(1000.),       
+	     _w_dxyPhiNormMax(1.8), _w_dzPhiNormMax(1.8),   
+	     _w_dxyEtaNormMax(1.8), _w_dzEtaNormMax(1.8) {}   
+
+  // getter methods
+
+  std::pair <float,float> get_dxyPhiMax() const {
+    std::pair <float , float> res(_m_dxyPhiMax,_w_dxyPhiMax);
+    return res;
```
Comment (ghellwig) [inline Alignment/OfflineValidation/macros/FitPVResiduals.C]: @mmusich just a minor comment: you could save some typing if you replace these two lines by
```
return std::make_pair(_m_dxyPhiMax,_w_dxyPhiMax);

**Review by ghellwig — APPROVED** [review]
Looks fine for me! The minor comment can be left for possible future updates.

**Comment by ghellwig** [comment]
+1
provided jenkins agrees

**Comment by davidlange6** [comment]
+1

**Pull request merged**